src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $main-card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: scale 0.3s;
+
+  &:hover {
+    scale: 1.2;
```

Comment:
The `scale` property is not a standalone CSS property. Use `transform: scale(1.2);` to apply a scaling transformation on hover.

Reply:
Fix it 
<img width="951" alt="image" src="https://github.com/user-attachments/assets/414e1ef6-2e79-467b-9dba-e7377174f84a">